doc/source/reference/credits.rst

```diff
@@ -259,7 +259,7 @@ Matthew Palathingal
 Kelly Wang
 
 * Enabled NeighborList indexing.
-* Added methods ``compute_distances`` and ``compute_all_distances`` to Box.
+* Added methods ``compute_distances``, ``compute_all_distances``, and ``crop`` to Box.
```

Comment:
Give yourself a new line since this is a separate PR. 😄
```suggestion
* Added methods ``compute_distances`` and ``compute_all_distances`` to Box.
* Added ``crop`` feature to Box.
```